ckanext/stadtzh_losdharvest/profiles.py

```diff
@@ -108,6 +108,9 @@ def parse_dataset(self, dataset_dict, dataset_ref):
         dataset_dict["tags"] = [
             {"name": munge_tag(tag)} for tag in self._keywords(dataset_ref)
         ]
+        # set fixed tag
+        dataset_dict["tags"].append("lod")
```

Comment:
The tag should be a dict, like so:

```suggestion
        dataset_dict["tags"].append({"name": "lod"})
```